package.json

```diff
@@ -56,13 +57,15 @@
   "devDependencies": {
     "@architect/architect": "^10.10.0",
     "@aws-sdk/client-cognito-identity-provider": "3.188.0",
+    "@aws-sdk/client-ses": "3.188.0",
```

Comment:
Can we use client-sesv2, which we already use elsewhere in the code?

Reply:
Yep! Removing now